dbms/src/Interpreters/AsynchronousMetrics.cpp

```diff
@@ -128,13 +128,6 @@ static void calculateMaxAndSum(Max & max, Sum & sum, T x)
 
 void AsynchronousMetrics::update()
 {
-    {
-        if (auto mark_cache = context.getMarkCache())
-        {
-            set("MarkCacheBytes", mark_cache->weight());
-            set("MarkCacheFiles", mark_cache->count());
-        }
```

Comment:
keep these

Reply:
OK. Maybe we can display these metrics in panels of Grafana.

Reply:
ref https://github.com/pingcap/tiflash/issues/5042